test/Distributed/distributed_actor_isolation.swift

```diff
@@ -115,6 +114,9 @@ distributed actor DistributedActor_1 {
     await self.distHelloAsync()
     try self.distHelloThrows()
     try await self.distHelloAsyncThrows()
+
+    // Hops over to the global actor.
+    _ = await DistributedActor_1.staticMainActorFunc()
```

Comment:
Yay! Glad that just works, we'll need to double check if we reject distributed func getting a global actor -- I'm guessing that's nonsense hmmm